lib/active_storage/service/postgresql_service.rb

```diff
@@ -103,7 +103,8 @@ def generate_url(key, expires_in:, filename:, disposition:, content_type:)
         )
 
         generated_url = url_helpers.rails_postgresql_service_url(verified_key_with_expiration,
-          **url_options,
+          # **url_options,
```

Comment:
@krschacht had some time today to look into the exception again. It looks like the `**url_options` are `nil`. Replacing it with `only_path: true` solves the issue (at least in development). I am not sure if this is an acceptable fix.

Reply:
Oh, interesting. That makes sense. I can imagine the URL or host configuration may be different when running in workers. Probably we can just leave that change in place for always. It’s not like it ever needed a full URL.

Reply:
Pretty sure it will create a problem down the road for larger deployments with a CDN. But that's a future problem then.